AMW_angular/io/src/app/apps/apps-filter/apps-filter.component.ts

```diff
@@ -1,25 +1,24 @@
-import { Component, Input } from '@angular/core';
+import { ChangeDetectionStrategy, Component, EventEmitter, Input, OnInit, Output } from '@angular/core';
 
 import { NgSelectModule } from '@ng-select/ng-select';
 import { Release } from '../../settings/releases/release';
 import { FormsModule } from '@angular/forms';
-import { Output, EventEmitter } from '@angular/core';
 
 @Component({
   selector: 'app-apps-filter',
   standalone: true,
   imports: [FormsModule, NgSelectModule],
   templateUrl: './apps-filter.component.html',
+  changeDetection: ChangeDetectionStrategy.OnPush,
 })
 export class AppsFilterComponent {
   @Input() releases: Release[];
-
+  @Input() selectedReleaseId: number;
   @Output() filterEvent = new EventEmitter<{ filter: string; releaseId: number }>();
```

Comment:
I tried, but I am not quite sure how to do it, because I need it in AMW_angular/io/src/app/apps/apps-filter/apps-filter.component.html as `[(ngModel)]="selectedReleaseId`